pkg/cli/cli.go

```diff
@@ -491,15 +492,31 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 	// Local image
 	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
 		"Builds a CVD with image files built locally, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images and cvd-host-packages.tar.gz")
-	localImgMutuallyExFlags := []string{
+	// Local artifacts flag
+	d := "Path to local artifacts config file in toml format. For example:" +
+		"\n`````" + `
+Bootloader="PATH/bootloader"
+
+CVDHostPackageTar="PATH/cvd-host_package.tar.gz"
+
+[Images]
+boot = "PATH/boot.img"
+init_boot = "PATH/init_boot.img"
+super = "PATH/super.img"
+userdata = "PATH/userdata.img"
+` + "```"
+	create.Flags().StringVar(&createFlags.LocalArtifacts, localArtifactsFlag, "", d)
+	localBuildMutuallyExFlags := []string{
```

Comment:
nit: This name is a mouthful and doesn't make the code any clearer. Consider having two lists: one for remote build flags and one for local build flags; and then two nested loops to mark them mutually exclusive.

Reply:
good point, done